ASCIIToSVG.php

```diff
@@ -304,10 +306,44 @@ public function popGroup() {
     $this->curGroup = array_pop($this->groupStack);
   }
 
+    /*
+    AAVZZ
+    Since SVG has no notion of layers, we emulate them by changing the order
+    of boxes. This involves tagging and reordering. Tagging is done in `addObject',
+    reordering in `rearrangeBoxes'. We tag a box with a layer prefix, like `a' (default),
+    `b', `c' and so on. So, the sequence numbers will be something like
+    a0, a1, b2, a3, a4, b5, c6 ...
+    Once all the boxes are parsed, we need to rearrange them like
+    a0, a1, a3, a4, b2, b5, c6 ...
+    and change the indices to be numeic (we rely on them being numeric!!)
+    XXX Some text lines belonging to overlying box are placed in underlying box
+    XXX and do not show up. Further investigation is necessary to place them correctly.
+    XXX This issue is not introduced with this patch.
+  */
```

Comment:
Can you please make this comment block follow the style of other comment blocks in this file? For example:

```
/* 
 * Since SVG has no notion of layers...
```

Please remove your github username from the comment. It isn't informative to the code or the comment and this ends up being distracting when reading the code.

Please only use one instance of `XXX` in the comment. E.g.

```
 * XXX: Some text lines ...
 * and do not show up...
 */
```

If you intend on solving this problem, please feel free to change the `XXX:` to `TODO(aavzz):`

Please also fix the contents of this comment based on the discussion about indexing below.